packages/schema/bind/src/bindings/plugin-ts/index.ts

```diff
@@ -52,10 +52,10 @@ export function generateBinding(
     });
   };
 
-  const queryContext = typeInfo.queryTypes.find((def: QueryDefinition) => {
+  const queryContext = typeInfo.moduleTypes.find((def: QueryDefinition) => {
     return def.type === "Query";
   });
-  const mutationContext = typeInfo.queryTypes.find((def: QueryDefinition) => {
+  const mutationContext = typeInfo.moduleTypes.find((def: QueryDefinition) => {
```

Comment:
If only it were that simple :laughing: :sob:

This will not compile. You can test this building and testing the monorepo like described here: https://github.com/polywrap/monorepo#pre-reqs

If you're new to lerna monorepos, I'd recommend reading through this introduction: https://medium.com/@harshverma04111989/creating-monorepo-using-lerna-dd431d0db072

I'd recommend:
- Change the TypeInfo type definition (see `@web3api/schema-parse` project)
- Rename all other occurrences of this within the codebase (use an IDE with folder searching)
- Build & test all packages within the whole monorepo (starting with schema parse, compose, & bind projects)